src/interface.jl

```diff
@@ -10,6 +10,9 @@ set_current(o::Buffer) = set_current_buffer(o.client, o)
 set_current(o::Window) = set_current_window(o.client, o)
 set_current(o::Tabpage) = set_current_tabpage(o.client, o)
 
+cursor(c::Window) = get_cursor(c) + [0, 1]
```

Comment:
Nice! I wonder though if using a tuple of two Ints would be more idomatic here. I should improve typemapping to return tuples when the metadata prescribes fixed-length Array. Perhaps `cursor!(` for the setter to indicate that it mutates (sort-of) the first argument?